src/connectors/range/connectRange.js

```diff
@@ -100,8 +100,8 @@ export default function connectRange(renderFn) {
         }
 
         return {
-          min: Math.floor(min),
-          max: Math.ceil(max),
+          min: precision === 0 ? Math.floor(min) : min,
```

Comment:
What does it mean for a precision of 1 and there 3 digits after the `.` or `,`?

Reply:
When I begin to implement it, I thought it was a bit overkill.

Reply:
Well Isn't the purpose to bring consistency? And in this case, we only have consistency for precision = 0, right?

Reply:
But then I might not have a good idea of the complexity...

Reply:
Right, it will update it 🙃